src/style.css

```diff
@@ -1 +1,9 @@
-/* styles go here */
+.form {
```

Comment:
This is not styling for form, please change name accordingly e.g. ```form-item```. Same with info, something more informative would be better e.g. ```form-fieldset```